openbanking/deploy/setup-ce.sh

```diff
@@ -0,0 +1,1040 @@
+#!/bin/bash
+
+echo $(pwd)
+source deploy/ce_workspace.env
+
+function get_token {
+  ACCESS_TOKEN=$(curl --request POST \
+  --url https://$DOMAIN/$TENANT_ID/admin/oauth2/token \
+  --header "Authorization: Basic `echo -n $CLIENT_ID:$CLIENT_SECRET | base64`" \
+  --header 'Content-Type: application/x-www-form-urlencoded' \
+  --data grant_type=client_credentials | jq '.access_token')
+
+  ACCESS_TOKEN=`sed -e 's/^"//' -e 's/"$//' <<<"$ACCESS_TOKEN"`
+
+}
+
+function create_oauth_server {
+    echo "Creating workspace"
+
+    curl --request POST \
+  --url https://$DOMAIN/api/admin/$TENANT_ID/servers \
+  --header "Authorization: Bearer $ACCESS_TOKEN" \
+  --header 'Content-Type: application/json' \
+  --data '{
+    "tenant_id": "ce-apigeex-demo",
```

Comment:
Shouldn't the value of this be `$TENANT_ID`? There are also a few other places with this hardcoded ce-apigeex-demo.

Reply:
Good catch!